hexrd/ui/overlays/powder_diffraction.py

```diff
@@ -137,20 +152,14 @@ def generate_ring_points(self, tths, etas, panel, display_mode):
 
                 # append to list with nan padding
                 ring_pts.append(np.vstack([ang_crds, nans_row]))
-            elif display_mode in [ViewType.raw, ViewType.cartesian]:
-                # !!! must apply offset
-                xys_full = panel.angles_to_cart(ang_crds, tvec_c=self.tvec)
-
-                # !!! distortion
-                if panel.distortion is not None:
-                    xys_full = panel.distortion.apply_inverse(xys_full)
 
-                # clip to detector panel
-                xys, on_panel = panel.clip_to_panel(
-                    xys_full, buffer_edges=False
-                )
+            elif display_mode in [ViewType.raw, ViewType.cartesian]:
 
                 if display_mode == ViewType.raw:
+                    # !!! distortion
+                    if panel.distortion is not None:
+                        xys_full = panel.distortion.apply_inverse(xys_full)
```

Comment:
@joelvbernier So, is distortion only applied to raw and polar views? Looks like it isn't applied to Cartesian here.

Reply:
Good catch -- should only be applied for the raw.